### DISCUSSION TITLE
Feature suggestion: Make nonempty fields more visually explicit

### DISCUSSION THREAD

**Discussion by sergio9929** [pocketbase/pocketbase#3412] [Q&A]
Depending on the size of the screen or if I'm navigating the admin quickly, there are times when it's hard for me to see that gray dot.
Examples:
| Before | After |
| ------- | ----- |
| ![image](https://github.com/pocketbase/pocketbase/assets/29752861/486a8e7c-dcec-4bd3-b720-98371908bdfd) | ![image](https://github.com/pocketbase/pocketbase/assets/29752861/dc8bb83f-537e-404a-a947-0037472d5bec) |
| ![image](https://github.com/pocketbase/pocketbase/assets/29752861/65fe24e4-b5ff-4f55-9fd4-25bc58746ecc) | ![image](https://github.com/pocketbase/pocketbase/assets/29752861/07dd514c-2216-49ea-9a24-8d4da75ef908) |

I would suggest making it red, like the asterisk in the collection name, or making it bigger, or both. Maybe a "(nonempty)" text after the field name... idk.

**Comment by sergio9929** [comment]
A "nonempty" text has been added in pocketbase v0.19.0

**Reply by ganigeorgiev** [reply]
Yes, sorry, I forgot to mention it here.

**Reply by sergio9929** [reply]
Thank you for the change.
